src/api/app/controllers/webui/request_controller.rb

```diff
@@ -48,10 +48,26 @@ def add_reviewer
     redirect_to controller: :request, action: 'show', number: params[:number]
   end
 
-  def modify_review
-    review_params = params.slice(:comment, :by_user, :by_group, :by_project, :by_package)
-    request = BsRequest.find_by_number(params[:request_number])
+  def modify_review_set_request
+    review_params = params.slice(:comment, :by_user, :by_group, :by_project, :by_package, :review_id)
```

Comment:
If I understand correctly, the `by_*` parameters won't be needed anymore once the bento UI is dropped. I would add a comment for that.

Reply:
there is a TODO bento_only 2 lines below - but I extend it